packages/apps/composer-app/vite.config.ts

```diff
@@ -88,7 +88,11 @@ export default defineConfig({
       content: [
         resolve(__dirname, './index.html'),
         resolve(__dirname, './src/**/*.{js,ts,jsx,tsx}'),
-        resolve(__dirname, '../plugins/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../experimental/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../plugins/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../plugins/experimental/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../sdk/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../ui/*/src/**/*.{js,ts,jsx,tsx}'),
```

Comment:
We may benefit instead from putting these in `resolveContent.ts` as known peers. See: https://github.com/dxos/dxos/blob/main/packages/ui/react-ui-theme/src/config/resolveContent.ts#L10